cluster/addons/istio/auth/istio-auth.yaml

```diff
@@ -3,13 +3,15 @@ kind: Namespace
 metadata:
   name: kube-public
   labels:
+    addonmanager.kubernetes.io/mode: Reconcile
     istio-injection: disabled
 ---
 apiVersion: v1
 kind: Namespace
 metadata:
   name: kube-system
```

Comment:
This seems odd.  Does this add-on own this namespace? If not, how is it able to apply labels without removing labels set by other components or the admin?